packages/gamut/src/Interstitial/styles.module.scss

```
@@ -22,6 +22,14 @@
   margin-bottom: 1rem;
 }
 
+h1.title:focus {
```

Any reason these use `h1`?

```suggestion
.title:focus {
```

---

was following outdated pattern, going to move all of this into css-in-js anyway😎